src/main/java/teammates/logic/core/CommentsLogic.java

```diff
@@ -310,61 +329,63 @@ private void removeNonVisibleCommentsForInstructor(
-            List<CommentAttributes> commentsForCourse, StudentAttributes student, List<String> teammates,
-            List<String> sectionStudentsEmails, List<String> teamsInThisSection, HashSet<String> commentsVisitedSet,
-            List<CommentAttributes> comments) {
-        removeNonVisibleCommentsForSection(commentsForCourse, student, teammates, 
-                sectionStudentsEmails, teamsInThisSection, commentsVisitedSet, comments);
+    private void removeNonVisibleCommentsForCourse(List<CommentAttributes> commentsForCourse,                                                   StudentAttributes student,
+                                                   List<String> teammates, List<String> sectionStudentsEmails,
+                                                   List<String> teamsInThisSection, HashSet<String> commentsVisitedSet,
+                                                   List<CommentAttributes> comments) {
+        removeNonVisibleCommentsForSection(commentsForCourse, student, teammates,
+                                           sectionStudentsEmails, teamsInThisSection,
+                                           commentsVisitedSet, comments);
         
-        for(CommentAttributes c: commentsForCourse){
-            if(c.courseId.equals(student.course)){
-                if(c.recipientType == CommentRecipientType.COURSE) {
-                    removeGiverNameByVisibilityOptions(c, CommentRecipientType.COURSE);
+        for (CommentAttributes c : commentsForCourse) {
+            if (c.courseId.equals(student.course)) {
+                if (c.recipientType == CommentParticipantType.COURSE) {
+                    removeGiverNameByVisibilityOptions(c, CommentParticipantType.COURSE);
                 } else {
-                    removeGiverAndRecipientNameByVisibilityOptions(c, CommentRecipientType.COURSE);
+                    removeGiverAndRecipientNameByVisibilityOptions(c, CommentParticipantType.COURSE);
                 }
                 appendComments(c, comments, commentsVisitedSet);
             }
         }
     }
     
-    private void removeNonVisibleCommentsForSection(
-            List<CommentAttributes> commentsForSection,
-            StudentAttributes student, List<String> teammatesEmails, List<String> sectionStudentsEmails,
-            List<String> teamsInThisSection, HashSet<String> commentsVisitedSet, List<CommentAttributes> comments) {
+    private void removeNonVisibleCommentsForSection(List<CommentAttributes> commentsForSection,
+                                                    StudentAttributes student, List<String> teammatesEmails,
+                                                    List<String> sectionStudentsEmails,
+                                                    List<String> teamsInThisSection,
+                                                    HashSet<String> commentsVisitedSet,
+                                                    List<CommentAttributes> comments) {
```

Comment:
The 2nd line has two parameters, but the other lines have only one. For consistency sake, this can be a single-column stack. Alternatively, go for 8-space indent and avoid the single-column stack.